bindings/python/pymongoarrow/schema.py

```diff
@@ -62,10 +63,22 @@ def _normalize_mapping(mapping):
 
     def _get_projection(self):
         projection = {"_id": False}
-        for fname, _ in self.typemap.items():
-            projection[fname] = True
+        for fname, ftype in self.typemap.items():
+            if isinstance(ftype, ListType):
+                projection[fname] = self._get_field_projection_value(ftype.value_field.type)
+            else:
+                projection[fname] = self._get_field_projection_value(ftype)
         return projection
 
+    def _get_field_projection_value(self, ftype):
+        value = True
+        if isinstance(ftype, StructType):
```

Comment:
Do we also need to handle a list of lists?

Reply:
As discussed, lists of lists are projected the same as a list.